meson.build

```diff
@@ -0,0 +1,40 @@
+project(
+  'droidmedia',
+  'c',
+default_options : ['warning_level=1', 'buildtype=debugoptimized'],
+version : '0.20210608.1+jb54878.20210729075143.df4abcc-1'
```

Comment:
This probably should be set to something more simple.

Reply:
My fault. The spec uses meson rewrite feature to set the version, and I accedentally commited this after building locally. Fixed.